src/angular/ghettobox/ghettobox.scss

```diff
@@ -0,0 +1,96 @@
+@import '../styles/common';
+
+.sbb-ghettobox {
+  display: flex;
+  background-color: $sbbColorIron;
+  color: $sbbColorWhite;
+  padding: pxToRem(16);
+  justify-content: center;
+  align-items: flex-start;
+  font-size: pxToRem($sizeFontDefault);
+  line-height: pxToRem($sizeLineHeightDefault);
+
+  @include mq($from: tablet) {
+    padding: pxToRem(36) pxToRem(24) pxToRem(36) pxToRem(48);
+  }
+  @include mq($from: desktop) {
+    padding: pxToEm(40) pxToEm(56);
+  }
+  @include mq($from: desktop4k) {
```

Comment:
I know, ghettobox is standard only, but I think it would be better to wrap this and 5k with standard selector